contentcuration/contentcuration/catalog_settings.py

```diff
@@ -1,6 +1,8 @@
 # flake8: noqa
 from .production_settings import *  # noqa
 
+LANGUAGES += (("ar", ugettext("Arabic")),)  # noqa
```

Comment:
Highlighting this update as we are going to be focusing on Arabic translations for the catalog